improver_tests/wxcode/wxcode/test_WeatherSymbols.py

```diff
@@ -232,6 +232,36 @@ def setUp(self):
             if "vicinity" not in name and "sleet" not in name and "texture" not in name
         ]
 
+    def assertArrayAndMaskEqual(self, test, expected, **kwargs):
+        """
+        Runs assertArrayEqual and then checks that if a mask is present, it matches.
+
+        Args:
+            test (np.ndarray):
```

Comment:
Suggest commenting that this will be the output from a test case.

Reply:
Updated